app/controllers/api/v1/search_controller.rb

```diff
@@ -0,0 +1,15 @@
+class Api::V1::SearchController < ApiController
+  before_action :ensure_term
+
+  expose(:organizations) { Organization.estella_search(estella_options) }
+
+  private
+
+  def estella_options
+    params.permit(:term)
```

Comment:
This should also accept `size` and `page`  params for pagination. The latter is marshalled into a `from` offset as that's what ES works with (see [here](https://github.com/artsy/bearden/pull/308/files#diff-8b9642288245a75a3e8b37781a17c935L16)). I use `page` instead of `from` as that's the convention for pagination in all our core APIs.

Reply:
We can also add a spec for pagination params but it's pretty simple stuff.